Experimental middleware to create UI5 integration cards (#1668)
SAP/open-ux-tools: packages/cards-editor-middleware/README.md

#  EXPERIMENTAL `@sap-ux/cards-editor-middleware`

> A middleware to store generation card manifest and i18n text to application repository. Middlware is configured by `@sap-ux/cards-editor-config-writer`

Reviewer: ```suggestion
A middleware to generate a card manifest and i18n text in the project. Middleware is configured by `@sap-ux/cards-editor-config-writer`
```
